pkgs/by-name/is/iscc/package.nix

```diff
@@ -0,0 +1,58 @@
+{ stdenv
+, fetchurl
+, innoextract
+, runtimeShell
+, wineWow64Packages
+, lib
+}:
+
+let
+  version = "6.2.2";
+  majorVersion = builtins.substring 0 1 version;
+in
+stdenv.mkDerivation rec {
+  name = "iscc";
+  inherit version;
+  src = fetchurl {
+    url = "https://files.jrsoftware.org/is/${majorVersion}/innosetup-${version}.exe";
+    hash = "sha256-gRfRDQCirTOhOQl46jhyhhwzDgh5FEEKY3eyLExbhWM=";
+  };
+  nativeBuildInputs = [
+    innoextract
+    wineWow64Packages.stable
+  ];
+  unpackPhase = ''
+    innoextract $src
+  '';
+  dontBuild = true;
+  installPhase = ''
+    mkdir -p $out/bin
```

Comment:
Personal pet peeve: always quote shell variables when word splitting would break the code. (Yes, I know lots of other things would break if $out contained whitespace -- that's not the point -- the point is to write robust/correct code.)

Reply:
Not a bash guru. Happy to make those changes. Done.